architecture/core_objects/images_containers.adoc

```diff
@@ -0,0 +1,72 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+== Containers
+
+The basic units of OpenShift applications are called containers. Linux
+container technologies are lightweight mechanisms for isolating
+running processes so that they are limited to interacting with only
+their designated resources. Many application instances can be running
+in containers on a single host without visibility into each others'
+processes, files, network, and so on. Typically we expect that each will
+provide a single service (often called a "micro-service"), such as a
+web server or a database, though they can be used for arbitrary workloads.
+
+You can read further about container technology
+link:https://access.redhat.com/articles/1353593[here]. The Linux
+kernel has been incorporating capabilities for container
+technologies for years. More recently the Docker project has
+developed a convenient management interface for Linux containers
+on a host (link:https://www.docker.com/whatisdocker/[read more about
+Docker]). OpenShift and Kubernetes orchestrate Docker containers across
```

Comment:
This link could probably be moved to the "Docker project" bit before this, and the bit in the ()s removed.